src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/MagicWorkbench.java

```diff
@@ -43,8 +43,7 @@ public void onInteract(Player p, Block b) {
 
         BlockState state = PaperLib.getBlockState(dispenser, false).getState();
 
-        if (state instanceof Dispenser) {
-            Dispenser disp = (Dispenser) state;
+        if (state instanceof Dispenser disp) {
```

Comment:
It would be nice if you could apply the same `possibleDispenser` / `dispenser` naming pattern here like you did in the other cases.

Reply:
Yeah this one I did debate it internally but left it off as `locateDispenser()` ensures it is a dispenser. Ill apply it regardless :)